selfdrive/locationd/locationd.cc

```diff
@@ -365,7 +365,10 @@ int Localizer::locationd_thread() {
         VectorXd posGeo = this->get_position_geodetic();
         std::string lastGPSPosJSON = util::string_format(
           "{\"latitude\": %.15f, \"longitude\": %.15f, \"altitude\": %.15f}", posGeo(0), posGeo(1), posGeo(2));
-        params.put("LastGPSPosition", lastGPSPosJSON);  // TODO write async
+
+        std::thread([&params] (const std::string& gpsjson) {
```

Comment:
Does this still work if it takes long enough for `lastGPSPosJSON` to go out of scope and the memory to be reused?

Reply:
it seems that the std::thread call made a copy anyway, but I removed the reference to make it more clear